README.md

```diff
@@ -1 +1,23 @@
-# halcyonsim
\ No newline at end of file
+# halcyonsim
+
+Simulates a virtual world environment. With sun, moon and clouds moving in regular day cycles or static positions (depending on the environment settings).  The regions consist of the aformentioned environments and land, which can be subdivided just like in real life. Also buildings *(objects with seamless textures to give the desired look)* can be setup on the land, as well as in the sky on what are known as "sky platforms". Plus avatars *(the users' representation in-world)* and interactive objects that are scripted. Objects or buildings can have various scripts added to them which allows them to exhibit different functions (such as open doors, activate elevator, sit on furniture items with several different poses), as well as produce particles that resemble items in real life such as fire in a fireplace inside a building or campfire outside of buildings, and/or smoke from a smokestack on a building. 
+
+**HalcyonGrid** or **HalcyonSim** was a fork from the <a href="https://opensimulator.org" rel="nofollow" >OpenSimulator</a> Project in 2010, and is intended to be a reliable simulator. 
+
+## Virtual World where people may walk in a full 3D world.
+
+Halcyon World is a Virtual World with a full 3D environment,  written in C Sharp (C#), where you may move about with a character known as an avatar in a 3D World accessed with a 3D graphics application, commonly known as a "viewer", which is used to access/login to these virtual worlds (not a VR headset). The most common one currently is the <a href="https://firestormviewer.org" rel="follow">Firestorm viewer for OpenSim</a>.  *(the "OS" stands for 'OpenSim', and not 'Operating System')* also it is similar to <a href="https://SecondLife.com" rel="nofollow" >Second Life™</a>.  It allows users to "talk" to other people in-world through nearby chat, IM's, group chats and group IM's and sometimes in voice, depending on how the grid is setup and the permissions set on individual regions. Including offline IM's, sending the offline IM's to the user's registered email address. That is as much like a game as it gets. There is also the ability to build objects in-world known as primatives or "prims" in various shapes and sizes to make different types of linked or individual 3D objects. Additionally other 3D items known as "Mesh" (DEA files) can be imported which have been created/edited with a 3D object editor/creator such as the free program titled "<a href="https://store.steampowered.com/app/365670/Blender/" rel="follow" >Blender</a>", or exported from a world as DEA. 
```

Comment:
FYI Blender actually has its own website: https://www.blender.org/

Reply:
Not that we actually need to link to any 3D model editor in the first place...

Reply:
Ok, I see.. well I don't think there would be any problem linking to a website for an Open Source program.